app/static/openapi.yaml

```diff
@@ -517,6 +517,94 @@ paths:
         404:
           $ref: '#/components/responses/NotFound'
 
+  /resources/{id}/upvote:
+    put:
+      summary: Upvote Resource - Increment Number of Upvotes
+      description: |
+        Increments the `upvotes` property of a resource. If the user has already upvoted the resource, upvotes property will decrement since a user is limited to only one vote per resource.
+      tags:
+        - Upvote Resource
```

Comment:
This should be `- Update Resource`. The section for upvoting docs doesn't render because of this. Once this change is made, I think we're good to go, but I'm going to give it a quick once over again before we merge